solidity/test/maintainer/MaintainerProxy.test.ts

```diff
@@ -2057,124 +2057,6 @@ describe("MaintainerProxy", () => {
     })
   })
 
-  describe("submitMovingFundsCommitment", () => {
-    const walletDraft = {
-      ecdsaWalletID: ecdsaWalletTestData.walletID,
-      mainUtxoHash: ethers.constants.HashZero,
-      pendingRedemptionsValue: 0,
-      createdAt: 0,
-      movingFundsRequestedAt: 0,
-      closingStartedAt: 0,
-      pendingMovedFundsSweepRequestsCount: 0,
-      state: walletState.Unknown,
-      movingFundsTargetWalletsCommitmentHash: ethers.constants.HashZero,
-    }
-
-    // Just an arbitrary main UTXO with value of 26 BTC.
-    const mainUtxo = {
-      txHash:
-        "0xc9e58780c6c289c25ae1fe293f85a4db4d0af4f305172f2a1868ddd917458bdf",
-      txOutputIndex: 0,
-      txOutputValue: to1ePrecision(26, 8),
```

Comment:
We can now remove the import at the top.

Reply:
4c0301124afcfced638004a9b9b16ee038d5c6e6